addons/event-audit/common/src/main/java/org/commonjava/indy/event/audit/ISPNEventPublisher.java

```diff
@@ -0,0 +1,35 @@
+package org.commonjava.indy.event.audit;
+
+import org.commonjava.indy.subsys.infinispan.CacheHandle;
+import org.commonjava.indy.subsys.infinispan.CacheProducer;
+import org.commonjava.maven.galley.spi.cache.CacheProvider;
+import org.commonjava.propulsor.content.audit.FileEventPublisher;
+import org.commonjava.propulsor.content.audit.FileEventPublisherException;
+import org.commonjava.propulsor.content.audit.model.FileEvent;
+import org.commonjava.propulsor.content.audit.model.FileGroupingEvent;
+
+import javax.inject.Inject;
+
+public class ISPNEventPublisher implements FileEventPublisher
+{
+
+    @Inject
+    @FileEventCache
+    CacheHandle<String, FileEvent> fileEventCache;
+
+    @Inject
+    @FileGroupingEventCache
+    CacheHandle<String, FileGroupingEvent> fileGroupingEventCache;
+
+    @Override
+    public void publishFileEvent( FileEvent fileEvent ) throws FileEventPublisherException
+    {
+        fileEventCache.put( fileEvent.getEventId().toString(), fileEvent );
```

Comment:
Would it make sense to use putAsync()?

Reply:
The handler BasicCacheHandle does not have the interface putAsync() now.

Reply:
Ah, my mistake